### PR TITLE
Documentation: Fix typo "nest nest"->"nest" in README.md

### PR DESCRIPTION
**Pull request by christianhg**
## Description

*Describe the changes in this pull request*

Closes #

### Before

*Insert relevant screenshots/recordings/automated-tests*

### After

*Insert relevant screenshots/recordings/automated-tests*

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -172,7 +172,7 @@ Node Transforms and Command Listeners are called with an implicit `editor.update
 
 It is permitted to do nested updates, or nested reads, but an update should not be nested in a read
 or vice versa. For example, `editor.update(() => editor.update(() => {...}))` is allowed. It is permitted
-to nest nest an `editor.read` at the end of an `editor.update`, but this will immediately flush the update
+to nest a `editor.read` at the end of an `editor.update`, but this will immediately flush the update
```
Comment (etrepum) [inline README.md:175]:
```suggestion
to nest an `editor.read` at the end of an `editor.update`, but this will immediately flush the update
```

**Review by etrepum — DISMISSED** [review]

**Comment by etrepum** [comment]
"an" is still a bit more correct here unless you were to ignore the `editor.`. I would read this as "an editor read" not "a read".

**Comment by etrepum** [comment]
Thank you!

**Comment by christianhg** [comment]
> "an" is still a bit more correct here unless you were to ignore the `editor.`. I would read this as "an editor read" not "a read".

Ah, yes, that makes sense!

**Review by potatowagon — APPROVED** [review]

**Pull request merged**